server/src/main/java/org/opensearch/transport/ClusterConnectionManager.java

```diff
@@ -110,6 +112,22 @@ public void openConnection(DiscoveryNode node, ConnectionProfile connectionProfi
         internalOpenConnection(node, resolvedProfile, listener);
     }
 
+    @Override
+    public Set<DiscoveryNode> getNodesLeftInProgress() {
```

Comment:
Do we need to expose this to the caller ? 

Can we check during connect & remove from the list once disconnect is completed ?